src/app/stacks/create/components/sharedSchema.ts

```diff
@@ -1,9 +1,13 @@
-import { validateStackName } from "@/lib/stacks";
+import { validateStackNameNotInUse } from "@/lib/stacks";
 import { z } from "zod";
 
 export const stackNameSchema = z
 	.string()
 	.trim()
 	.min(1, "Stack name is required")
-	.max(255, "Stack name must be less than 255 characters")
-	.refine((name) => validateStackName(name), "Stack name is already in use");
+	.max(32, "Stack name must be less than 32 characters")
```

Comment:
How do you come to the conclusion of this?

<img width="498" height="336" alt="grafik" src="https://github.com/user-attachments/assets/c4bb5373-0317-4f67-bb88-aa81313a3d9b" />

I just checked the Swagger-Docs and the only restriction is the length of 255 characters that we had before 🤔